docs/docs/experiments/remote_taskfiles.md

```diff
@@ -48,6 +48,16 @@ tasks:
 and you run `task my-remote-namespace:hello`, it will print the text: "Hello
 from the remote Taskfile!" to your console.
 
+## Root remote Taskfiles
+
+You can also specify a remote Taskfile as the entrypoint for Task by using the
+`--taskfile` flag. This means you don't need to have any Taskfile defined
+locally at all. For example:
+
+```shell
+task --taskfile https://taskfile.dev hello
```

Comment:
We could consider having a proper Taskfile being served in the website, so copying and pasting would work 🙂 

Could be a GitHub or GitHub Gist URL as well...

```suggestion
task --taskfile https://taskfile.dev/taskfiles/hello.yml hello
```

Just an idea, does not need to be done on this PR...

Reply:
Not entirely clear what you mean by this

> We could consider having a proper Taskfile being served in the website

Is this not already complete in this PR? see https://github.com/go-task/task/pull/1347/files#diff-eea9e58f9afa47c7cdf1393b2d383a4713f91171f6603ced3937cacc09fe9376

I will merge for now and we can adjust later if need be.

Reply:
Oh, got it, it's already there in the root 🙂 

It just surprised me that it automatically fetches the `/Taskfile.yml` path. I thought the whole path would have to be explicit...

Reply:
This is what the `RemoteExists` function is doing. It's similar to what we do with local files where only a directory is given. Task will look around for a file that makes sense. I thought this would be nice for projects that want to use Taskfiles to do some setup work. e.g.

`task --taskfile https://myproject.org init` or something similar is a really nice, clean way of asking your devs to do run some setup task. Whereas `task --taskfile https://myproject.org/Taskfile.yml init` is a bit less tidy.

The same applies to the new `taskfile.dev/Taskfile.yml` file. If a user wants to check that their installation of Task works, they can now run `task --taskfile https://taskfile.dev` and it will reply with "Hello Task!". This is a bit less clunky than `task --taskfile https://taskfile.dev/Taskfile.yml`.

Happy to revisit this if you don't like the behaviour though.

Reply:
It's actually an interesting feature, we can keep it 👍 

I did quick review at the time, I so end up not paying attention to that function 🙂 

I just suggest to improve the documentation a bit to explain that this is a possibility, just like for local files.